src/Payload.cpp

```diff
@@ -43,6 +43,13 @@ std::string Payload::moveDataToString() {
   return data->moveToFbString().toStdString();
 }
 
+std::string Payload::cloneDataToString() const {
```

Comment:
I was matching the `moveDataToString` signature. Seems confusing to have one as a name with "move" and another with an operator overload.